Algorithms/DataCompression/huffman.cs

```diff
@@ -2,20 +2,21 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace DC5
+namespace Algorithms.DataCompression
 {
     internal class Huff
     {
-        public string Data { get; set; }
-        public int Frequency { get; set; }
-        public Huff RightChild { get; set; }
-        public Huff LeftChild { get; set; }
+        public string Data { get; }
+        public int Frequency { get; }
```

Comment:
Why does Frequency integer? Shouldn't it be a real number between 0 and 1?

Reply:
Frequency as "number of incidences/observations" per character.